instruqt-tracks/terraform-cloud-aws-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -l
+set -e
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
+export ORG=$(grep organization /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+export WORKSPACE=$(grep workspace /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+
+agent variable set TF_ORG $ORG
```

Comment:
Before we set the agent variable `TF_ORG` shouldn't we check to see that it has been changed from `YOUR_ORGANIZATION`?

Reply:
And error if it isn't?

Reply:
The logic is that the TF_ORG is set during exercise #2. We set the Instruqt variable here. 

If the user tries to use `YOUR_ORGANIZATION` the build will fail. In other words, we cannot reach this stage without specifying a valid TFC organization.

Reply:
We could though, couldn't we? User starts challenge 2 and immediately hits check. Or hits check after some typo prevents the `terraform` commands in the challenge from successfully completing.

Reply:
I understand. Check that the ORG has changed. That makes sense.